Buildings/Controls/OBC/FDE/DOAS/CoolingCoil.mo

```diff
@@ -0,0 +1,206 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block CoolingCoil "This block commands the cooling coil."
+
+
+  parameter Real erwDPadj(
+  final unit = "K",
+  final quantity = "TemperatureDifference") = 5
+  "Value subtracted from ERW supply air dewpoint.";
+
+  parameter CDL.Types.SimpleController controllerTypeDeh=Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller";
+  parameter Real kDeh=1
+    "Gain of controller";
+  parameter Real TiDeh=0.5
+    "Time constant of integrator block";
+
+  parameter CDL.Types.SimpleController controllerTypeRegOpe=Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller";
```

Comment:
Change descriptive comment to "Type of controller for regular operation PI block". Also applies to k and Ti. See comment about Td above